__tests__/app.test.js

```diff
@@ -29,7 +25,20 @@ it("Responds with the correct keys", () => {
         });
       });
   });
+});
+
+describe("GET/api/endPoints", () => {
+  it("Responds with a descriptive list of valids endPoints ", () => {
+    return request(app)
+      .get("/api")
+      .expect(200)
+      .then((res) => {
+        expect(res.body).toEqual(endPoints);
```

Comment:
Endpoints should come in in an object on a key of 'endpoints', rather than directly on the 'body'.